base/pkg/read.jl

```diff
@@ -3,18 +3,29 @@
 module Read
 
 import ...LibGit2, ..Cache, ..Reqs, ...Pkg.PkgError, ..Dir
-using ..Types
+using ..Types, Base.Filesystem
 
-readstrip(path...) = strip(readstring(joinpath(path...)))
+isgitrepo(pkg) =
+    ispath_casesensitive(joinpath(pkg, ".git"))
```

Comment:
If we want to check whether `pkg` is capitalized correctly here, we should do `isdir_casesensitive(pkg) && ispath(pkg, ".git")`